docs/content/docs/ops/debugging/flame_graphs.md

```diff
@@ -46,6 +46,12 @@ Starting with Flink 1.13, Flame Graphs are natively supported in Flink. In order
 Operator's On-CPU Flame Graph
 {{% /center %}}
 
+{{< hint info >}}
+**Warn:** 
+Any measurement process in and of itself inevitably affects the subject of measurement (see the [double-split experiment](https://en.wikipedia.org/wiki/Double-slit_experiment)). CPU stack traces sampling is no exception. In order to prevent the unintended impact of such a measurement on a production environment, Flame Graphs are currently available as an opt-in feature. To enable it, you'll need to set the [`rest.flamegraph.enabled: true`]({{< ref "docs/deployment/config">}}#rest-flamegraph-enabled) in `conf/flink-conf.yaml`. We recommend enabling it in development and pre-production environments but treat it as an experimental feature in the production context.
```

Comment:
```suggestion
Any measurement process in and of itself inevitably affects the subject of measurement (see the [double-split experiment](https://en.wikipedia.org/wiki/Double-slit_experiment)). Sampling CPU stack traces is no exception. In order to prevent the unintended impact of such a measurement on a production environment, Flame Graphs are currently available as an opt-in feature. To enable it, you'll need to set [`rest.flamegraph.enabled: true`]({{< ref "docs/deployment/config">}}#rest-flamegraph-enabled) in `conf/flink-conf.yaml`. We recommend enabling it in development and pre-production environments, but you should treat it as an experimental feature in production environments.
```